theano/tensor/blas_headers.py

```diff
@@ -984,7 +981,11 @@ def blas_header_text():
                     }
                     """)
 
-    return (header % {'const': const}) + gemm_code
+    return header + blas_code
+
+
+if not config.blas.ldflags:
+    print('Using NumPy C-API based implementation for BLAS functions.')
```

Comment:
Could it be possible to use the `logging` module here instead?

Reply:
Done!